caim_base/templates/animal/view.html

```diff
@@ -83,31 +84,75 @@ <h2>{{ animal.name }}</h2>
-          <h4>Special Needs</h4>
-          <p>
-            {{ animal.special_needs | urlizetrunc:60 | linebreaks }}
-          </p>
-          {% endif %}
-
-          {% if animal.vaccinations_notes %}
-          <h4>Vaccination Notes</h4>
-          <p>
-            {{ animal.vaccinations_notes | urlizetrunc:60 | linebreaks }}
-          </p>
-          {% endif %}
-
-          {% include './comments.html' %}
-
-
+          </div>
+          <div class="row gx-5 gy-3">
+            <div class="col-lg animal-detail-panel">
+              <h5 class="black-header mb-3">Behavior</h5>
+              <div>
+                <div>
+                  <img src="{% static 'img/icon-dog.svg' %}">
+                  with dogs:
+                </div>
+                <div>{{ animal.behaviourDogsText }}</div>
+              </div>
+              <div>
+                <div>
+                  <img src="{% static 'img/icon-cat.svg' %}">
+                  with cats:
+                </div>
+                <div>{{ animal.behaviourCatsText }}</div>
+              </div>
+              <div>
+                <div>
+                  <img src="{% static 'img/icon-kid.svg' %}">
+                  with kids:
+                </div>
+                <div>{{ animal.behaviourKidsText}}</div>
+              </div>
+            </div>
+            <div class="col-lg animal-detail-panel">
+              <h5 class="black-header mb-3">Medical Notes</h5>
+              <div class="d-flex justify-content-between">
+                <div>
+                  <img src="{% static 'img/icon-spayneuter.svg' %}">
+                  Spayed / Neutered:
+                </div>
+                <div>{{ animal.is_spayed_neutered | yesno | capfirst }}</div>
+              </div>
+              <div class="d-flex justify-content-between">
+                <div>
+                  <img src="{% static 'img/icon-vaccination.svg' %}">
+                  Vaccinations up to date:
+                </div>
+                <div>{{ animal.is_vaccinations_current | yesno | capfirst }}</div>
+              </div>
+              {% if animal.vaccinations_notes %}
+                <p>
+                  {{ animal.vaccinations_notes | urlizetrunc:60 | linebreaks }}
+                </p>
+              {% endif %}
+              <div class="d-flex justify-content-between">
+                <div>
+                  <img src="{% static 'img/icon-specneeds.svg' %}">
+                  Special needs:
+                </div>
+                <div>{{ animal.is_special_needs | yesno | capfirst }}</div>
+              </div>
+              {% if animal.special_needs %}
+                <p>
+                  {{ animal.special_needs | urlizetrunc:60 | linebreaks }}
+                </p>
+              {% endif %}
+            </div>
+          </div>
         </div>
       </div>
+      {% include './comments.html' %}
     </div>
-    <div class="col-md-4">
+    <div class="col-md-4 mt-3 mt-md-0">
```

Comment:
I'm currently at aspiring CSS noob status, what does this do?

Reply:
It's a way of [applying styles for different screen sizes](https://getbootstrap.com/docs/5.0/layout/breakpoints/). Since Bootstrap is mobile-first, `mt-3` applies the top margin of `$spacer`, and then `mt-md-0` gets rid of the top margin for screen widths larger than the `md` breakpoint (768px).

In this case, we want top margin for the mobile design (<768px), since we want some spacing between the comment box and the AWG card. We don't want any top margin for the desktop view, though, since it would break the alignment with the animal details card.